lbrynet/daemon/Daemon.py

```diff
@@ -951,192 +988,640 @@ def jsonrpc_commands(self):
+                "total_input": (int) total of inputs in dewies,
+                "total_output": (int) total of outputs in dewies(input - fees),
+                "txid": (str) txid of the transaction,
+            }
+
+            If sending a claim tip:
+            (dict) Dictionary containing the result of the support
+            {
+                txid : (str) txid of resulting support claim
+                nout : (int) nout of the resulting support claim
+                fee : (float) fee paid for the transaction
+            }
         """
 
-        # the check_locked() in the if statement is needed because that is what sets
-        # the wallet_unlocked_d deferred ¯\_(ツ)_/¯
-        if not self.wallet.check_locked():
-            d = self.wallet.wallet_unlocked_d
-            d.callback(password)
-            result = yield d
+        amount = self.get_dewies_or_error("amount", amount)
+        if not amount:
+            raise NullFundsError
+        elif amount < 0:
+            raise NegativeFundsError()
+
+        if address and claim_id:
+            raise Exception("Given both an address and a claim id")
+        elif not address and not claim_id:
+            raise Exception("Not given an address or a claim id")
+
+        if address:
+            # raises an error if the address is invalid
+            decode_address(address)
+
+            reserved_points = self.wallet_manager.reserve_points(address, amount)
+            if reserved_points is None:
+                raise InsufficientFundsError()
+            result = yield self.wallet_manager.send_points_to_address(reserved_points, amount)
+            self.analytics_manager.send_credits_sent()
         else:
-            result = True
-        response = yield self._render_response(result)
-        defer.returnValue(response)
+            validate_claim_id(claim_id)
+            result = yield self.wallet_manager.tip_claim(claim_id, amount)
+            self.analytics_manager.send_claim_action('new_support')
+        defer.returnValue(result)
 
     @requires(WALLET_COMPONENT, conditions=[WALLET_IS_UNLOCKED])
-    def jsonrpc_wallet_decrypt(self):
+    # @AuthJSONRPCServer.deprecated("account_fund"), API has changed as well, so we forward for now
+    # marked as deprecated in changelog and will be removed after subsequent release
+    def jsonrpc_wallet_prefill_addresses(self, num_addresses, amount, no_broadcast=False):
         """
-        Decrypt an encrypted wallet, this will remove the wallet password
+        Create new UTXOs, each containing `amount` credits
 
         Usage:
-            wallet_decrypt
+            wallet_prefill_addresses [--no_broadcast]
+                                     (<num_addresses> | --num_addresses=<num_addresses>)
+                                     (<amount> | --amount=<amount>)
 
         Options:
-            None
+            --no_broadcast                    : (bool) whether to broadcast or not
```

Comment:
why the negative form and not just `broadcast`?

Reply:
So as not to make any changes to api signatures I guess. This was how it was in the old version